src/Core/Environment.php

```diff
@@ -151,4 +166,92 @@ public static function getTimeLimitMax()
     {
         return static::$timeLimitMax;
     }
+
+    /**
+     * Get value of environment variable
+     *
+     * @param string $name
+     * @return mixed Value of the environment variable, or null if not set
+     */
+    public static function getenv($name)
```

Comment:
Can we make this camel case, so getEnv()?

Reply:
This api mirrors the old global `getenv`, so I've opted to maintain the semantics as close as possible, including case name.

Reply:
We can make it getEnv